content/open-firmware-systems.md

```diff
@@ -55,5 +56,8 @@ The source code for our Open EC can be found [here](https://github.com/system76/
 | Lemur Pro (lemp9 and newer) | Yes | Yes |
 | Oryx Pro (oryp5 and older) | No | No |
 | Oryx Pro (oryp6 and newer) | Yes | Yes |
+| Pangolin (pang12 and newer) | No | No |
```

Comment:
Oh also, this could probably be pang10 and newer. Panp9 is probably too old to be relevant, but I'm sure there are pang10/11's still out there kickin' around.